### PR TITLE
Append PDF text to links for accessibility

### PR DESCRIPTION
**Pull request by robertvrabel**
Example "Download" becomes "Download (pdf)":

Screen Shot 2019-04-24 at 11 56 44 AM

### CODE REVIEW

File: resources/js/modules/pdf.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import 'accordion/src/accordion.js';
```
Comment (nickdenardis) [inline resources/js/modules/pdf.js]: Is this needed?

**Comment by nickdenardis** [comment]
For reference this (pdf) does get read by NVDA:

targetprocess-screen-capture

File: resources/js/modules/pdf.js (hunk 1 above)
Comment (robertvrabel) [reply resources/js/modules/pdf.js]: It is not, I'll remove!

**Comment by robertvrabel** [comment]
@nickdenardis Yea we can add it to the styleguide, here is the PR. Let me know your thoughts on the wording. https://github.com/waynestate/base-site/pull/311

**Pull request closed without merging**